tools/mc-java/src/test/java/io/spine/tools/mc/java/annotation/check/FieldAnnotationCheck.java

```diff
@@ -28,59 +28,54 @@
 
 import com.google.protobuf.Descriptors.FieldDescriptor;
 import io.spine.code.proto.FieldName;
-import org.checkerframework.checker.nullness.qual.Nullable;
 import org.jboss.forge.roaster.model.JavaType;
 import org.jboss.forge.roaster.model.TypeHolder;
+import org.jboss.forge.roaster.model.VisibilityScoped;
 import org.jboss.forge.roaster.model.impl.AbstractJavaSource;
 import org.jboss.forge.roaster.model.source.JavaClassSource;
 import org.jboss.forge.roaster.model.source.JavaSource;
-import org.jboss.forge.roaster.model.source.MethodSource;
-
-import java.util.Optional;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static io.spine.code.java.SimpleClassName.ofBuilder;
-import static io.spine.tools.mc.java.annotation.check.Annotations.findInternalAnnotation;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class FieldAnnotationCheck implements SourceCheck {
+public class FieldAnnotationCheck extends SourceCheck {
```

Comment:
Can we please make it `final`? It'd be also great to have some docs for the check.